project/src/views/new-evaluation/newevaluation.js

```diff
@@ -0,0 +1,412 @@
+        <input class="inputs-date inputs-date-skin" type="date">
+    </div>
+    
+    `;
+}
+
+const TechnicalLevelPicker = function() {
+    return `
+        <div class="content-box">
+            <table class="content-box-tbl content-box-tbl-skin">
+            <caption class="content-box-tbl-tl content-box-tbl-tl-skin">Technical Level</caption>
+            <tr>
+                <th class="tbl-head tbl-head-skin">Trainee</th>
+                <th class="tbl-head tbl-head-skin">Junior</th>
+                <th class="tbl-head tbl-head-skin">Middle</th>
+                <th class="tbl-head tbl-head-skin">Senior</th>
+            </tr>
+            <tr class="second-half-of-table second-half-of-table-skin">
+                <td><input type="radio"></td>
+                <td><input type="radio"><input type="radio"><input type="radio"></td>
+                <td><input type="radio"><input type="radio"><input type="radio"></td>
+                <td><input type="radio"><input type="radio"><input type="radio"></td>
+            </tr>
+            </table>
+        </div>
+    `;
+}
+
+const Textarea = function() {
+    return `
+    <div class="content-box-same">
+        <h2 class="content-box-head content-box-head-skin">Should the candidate be hired?</h2>
+        <textarea class="content-box-text content-box-text-skin" rows="2">The type of project that is suitable for the candidate Is guidance required for the candidate</textarea>
+    </div>
+
+    <div class="content-box-same">
+        <h2 class="content-box-head content-box-head-skin">General Impression</h2>
+        <textarea class="content-box-text content-box-text-skin" required>*required</textarea>
+    </div>
+
+    <div class="content-box-same">
+        <h2 class="content-box-head content-box-head-skin">Workflow, Leadership &amp; Soft Skills</h2>
+        <textarea class="content-box-text content-box-text-skin" rows="2">Describe the environment in which the candidate works.
+        Describe any guidance or management experience</textarea>
+    </div>
+    `;
+}
+
+const DropdownSelector = function(options = {}) {
+
+  const dropDownOptions = [];
+  let val = 0;
+
+
+  options.options.forEach(function(el) {
+    const generatedEl = `<option value="${val}">${el}</option>`;
+    val ++;
+    dropDownOptions.push(generatedEl);
+  });
+
+  const dropdownEl = dropDownOptions.join('');
+
+  const aux = options.label.split(' ').join('_');
+  return `
+    <li>
+      <label class="firstLi" for="${aux}">${options.label}</label>
+      <select name="classEvalOptions" id="${options.label}">
+      ${dropdownEl}
+      </select> 
+    </li>`;
+}
+
+const TechnicalAreaPicker = function(options = {}) {
+    return `
+  
+    <div class="content-box-fields">
+    <fieldset class="content-box-evaluations content-box-evaluations-skin">
+      <legend class="field-title field-title-skin">OOP, Design Patterns</legend>
+        <ul>
+            ${DropdownSelector({
```

Comment:
iterate to create tge ul contents instead of duplicating calls to DropdownSelector(hint use a map over an array of objects)